src/spesh/pea.c

```diff
@@ -365,7 +365,7 @@ static MVMSpeshFacts * get_shadow_facts_c(MVMThreadContext *tc, GraphState *gs,
 static MVMSpeshFacts * create_shadow_facts_h(MVMThreadContext *tc, GraphState *gs, MVMuint16 idx) {
     MVMSpeshFacts *facts = get_shadow_facts_h(tc, gs, idx);
     if (!facts) {
-        ShadowFact sf;
+        ShadowFact sf = {0};
```

Comment:
Careful here. `ShadowFact` embeds a large struct `MVMSpeshFacts` and I don't know how that one works :shrug: